core/rawdb/schema.go

```diff
@@ -110,6 +110,13 @@ var (
 	firstQueueIndexNotInL2BlockPrefix = []byte("q")  // firstQueueIndexNotInL2BlockPrefix + L2 block hash -> enqueue index
 	highestSyncedQueueIndexKey        = []byte("HighestSyncedQueueIndex")
 
+	// Scroll rollup event store
+	rollupEventSyncedL1BlockNumberKey = []byte("LastRollupEventSyncedL1BlockNumber")
+	batchChunkRangesPrefix            = []byte("bcr")
+	batchMetaPrefix                   = []byte("bm")
+	finalizedL2BlockNumberKey         = []byte("finalized")
+	blockNumberToBatchIndexPrefix     = []byte("bn2bi")
```

Comment:
I'm now concerned that these will interfere with `blockBodyPrefix`, let's use different (non-overlapping) prefixes here.

Reply:
added an `R-` prefix in [88eed96](https://github.com/scroll-tech/go-ethereum/pull/515/commits/88eed967ad470daf15d7f952355e296d31895a01)